spec/helpers/ems_cloud_helper/textual_summary_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "spec_helper"
+
+describe EmsCloudHelper do
+  before do
+    controller.send(:extend, EmsCloudHelper)
+    self.class.send(:include, EmsCloudHelper)
+  end
+
+  context "textual_instances" do
+    def role_allows(_)
+      true
+    end
```

Comment:
We're calling it wrong. (But like the above, @AparnaKarve is following all the existing precedent.) I'll fix it.